### DISCUSSION TITLE
Add support for Ovis 1.6 small vision model?

### DISCUSSION THREAD

**Discussion by Jonseed** [ggml-org/llama.cpp#10169] [Ideas]
Performance: With just 10B parameters, Ovis1.6-Gemma2-9B leads the OpenCompass benchmark among open-source VLMs within 30B parameters. There is also a Llama 3.2 3B version.

And now there are also 4-bit quants available of both:

https://huggingface.co/AIDC-AI/Ovis1.6-Gemma2-9B-GPTQ-Int4

https://huggingface.co/AIDC-AI/Ovis1.6-Llama3.2-3B-GPTQ-Int4

**Comment by matheusborges** [comment]
Ovis1.6-Gemma2-9B
This model would be perfect for a local multimodal chatbot...
